src/Explorer/Tabs/Tabs.tsx

```diff
@@ -27,6 +28,7 @@ interface TabsProps {
 
 export const Tabs = ({ explorer }: TabsProps): JSX.Element => {
   const { openedTabs, openedReactTabs, activeTab, activeReactTab, networkSettingsWarning } = useTabs();
+  const queryCopilotState = useQueryCopilotState();
```

Comment:
You should move this hook inside `QueryCopilotTab.tsx`.

Reply:
I don't think that would be possible due to above comment (or I am missing something)